src/AddExerciseForm.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Button, FormControl, FormGroup, TextField,
+} from '@material-ui/core';
+import {
+  ChangeEvent, useState, FC,
+} from 'react';
+import useLocalStorage from './hooks/useLocalStorage';
+
+const AddExerciseForm: FC = () => {
+  const [formValues, setFormValues] = useState({ description: '', sets: '', reps: '' });
```

Comment:
Actually I think these should be three separate `useState` hooks, one for description, one for sets, and one for reps.

Reply:
This means your `onInputChange` will actually be split into `onDescriptionChange`, `onSetsChange`, etc, which can probably just be added inline to the text fields.